README.md

```diff
@@ -109,6 +109,13 @@ ks param set kubeflow-core usageId $(uuidgen)
 ks apply default -c kubeflow-core
 ```
 
+If the deployment is on a standalone cluster i.e. without access to a loadbalancer, then we need these two lines to access the Jupyter notebook: 
```

Comment:
Can you clarify what you mean by standalone cluster? How are you connecting to the nodes? Are you behind a firewall on a corporate network?

Reply:
ok I should have clarified: on a single K8 node that is on-premise (not cloud).